.github/workflows/run-tests.yaml

```diff
@@ -26,6 +26,11 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+      - name: Delete test models older than 72 hours
+        run: |
+          export HF_TAG=$(python -c "import os; print(os.environ.get('GITHUB_HEAD_REF') or os.environ.get('GITHUB_REF_NAME'))")
+          python -c "from huggingface_hub import delete_repo; delete_repo(token='$BLOOM_TESTING_WRITE_TOKEN', \
+          name='test-bloomd-560m-$HF_TAG', organization='bloom-testing')" || true
```

Comment:
Please consider moving that to a small Python script in the repo. Having a Bash command running Python, whose results are passed to another Bash command running Python, is not good.